week01/Trapping Rain Water.cpp

```diff
@@ -0,0 +1,27 @@
+class Solution {
+public:
+    int trap(vector<int>& height) {
+        int leftCurrentIndex = 0, rigthCurrentIndex = height.size()-1, leftMax = 0, rightMax= 0, result = 0;
```

Comment:
Unnecessary long variable naming here. `leftCurrentIndex` does not actually carry any significant weight than `L`.